estimate-web/src/components/UserStoryCommentList.jsx

```diff
@@ -39,12 +43,12 @@ class UserStoryCommentList extends Component {
                     <React.Fragment key={comment.id}>
                         <Persona
                             key={comment.createdBy.id}
-                            size={PersonaSize.size40}
+                            size={PersonaSize.size32}
                             imageUrl={comment.createdBy.imageUrl}
                             hidePersonaDetails
                         />
                         <div className="user-story-comment">
-                            <Label>
+                            <Label style={{ fontSize: FontSizes.smallPlus }}>
```

Comment:
Shouldn't we define this kind of thing in the css sheets?

Reply:
Office UI Fabric overrides these things when I define them in CSS myself :/
It's either this here (will leave for now) or explicitly pulling down their CSS and applying their own classes.
See `implementation` here: https://developer.microsoft.com/en-us/fabric#/styles/web/typography